config/projects/mozci.yml

```
@@ -241,7 +241,7 @@ mozci:
       description: Run mozci classification tasks for new pushes
       owner: mcastelluccio@mozilla.com
       emailOnError: true
-      schedule: ['*/60 * * * *'] # every hour
+      schedule: ['*/10 * * * *'] # every hour
```

```suggestion
      schedule: ['*/10 * * * *']
```
we can remove that redundant and misleading comment also